src/server.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// Config
+const express = require('express');
+const body_parser = require('body-parser');
+const fs = require('fs');
```

Comment:
Why is the File System in use here? Also it is improper to require some package and not use it.